samples/azure-service-bus-netstandard/native-integration/sample.md

```diff
@@ -48,7 +48,7 @@ snippet: NecessaryHeaders
 
 NOTE: The `NServiceBus.EnclosedMessageTypes` property must contain the the fully-qualified name of the type expected by the NServiceBus endpoint.
 
-The message itself is defined as an `IMessage` in the `Shared` project.
+The message itself is defined as message using [conventions](/nservicebus/messaging/conventions.md) in the `Receiver` project.
 
```

Comment:
Shall we add a snippet here with 
`endpointConfiguration.Conventions().DefiningMessagesAs(type => type.Name == "NativeMessage");`?

Reply:
good point, probably yes we should

Reply:
I'm not convinced we should. The focus of sample is about ASBS, not conventions. If we want to _link_ to conventions, we could do so w/o adding extra code snippets and stealing the focus from the main objective. Which is ASBS transport.

For example:
_Note: the sample is using Message Conventions. More about this feature [here](https://docs.particular.net/nservicebus/messaging/conventions)_